packages/react/src/components/OverflowMenu/_overflow-menu.scss

```diff
@@ -3,9 +3,3 @@
 button.#{$prefix}--overflow-menu {
   background: none;
 }
-
-.#{$prefix}--overflow-menu-options[data-floating-menu-direction] {
-  &:after {
-    width: $spacing-09;
-  }
-}
```

Comment:
In https://deploy-preview-3141--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-table-statefultable--stateful-example-with-expansion-max-pages-and-column-resize the overflow menu "spacer" is too large, but I _think_ that's because my PR where size is passed from the table down to the RowActionsCell hasn't been merged into this PR, right? 

![Screen Shot 2021-12-13 at 5 26 13 PM](https://user-images.githubusercontent.com/609466/145898911-7ae905a1-a940-4a16-a1f8-088fd081fbd0.png)

Reply:
Yes, that issue was created to prevent this, and now that it is merged the problem is solved.